src/main/java/io/rebloom/client/ClusterClient.java

```diff
@@ -96,12 +107,16 @@ public ClusterClient(Set<HostAndPort> jedisClusterNode, int connectionTimeout, i
      * Note that if a filter is not reserved, a new one is created when {@link #add(String, byte[])}
      * is called.
      */
-    public boolean createFilter(String name, long initCapacity, double errorRate) {
-        return (new JedisClusterCommand<Boolean>(this.connectionHandler, this.maxAttempts) {
-            public Boolean execute(Jedis connection) {
+    public void createFilter(String name, long initCapacity, double errorRate) {
```

Comment:
why did you change the method signature? It's a backward breaking change

Reply:
It's same with Client.java.

Reply:
it will create backward issue for existing users

Reply:
ok, I'm revert it.